app/javascript/src/utilities.js

```diff
@@ -281,12 +281,14 @@ function maxWidth($collection) {
 
 
 /* Get the difference between two numbers
+ * defaults to returning an absolute value
  **/
-function difference(a, b) {
-  if(a > b)
+function difference(a, b, abs=true) {
+  if(abs) {
+    return Math.abs(a - b);
+  } else {
     return a - b;
-  else
-    return b - a;
+  }
 }
```

Comment:
* Simplify method to use JS built-in `Math.abs` 
* Add param to indicate if we want a signed result.